MantidQt/CustomInterfaces/src/Muon/MuonAnalysisResultTableTab.cpp

```diff
@@ -497,40 +497,47 @@ void MuonAnalysisResultTableTab::populateLogsAndValues(
     // Get log information
     std::string wsName = fittedWsList[i].toStdString();
     auto ws = retrieveWSChecked<ExperimentInfo>(wsName + WORKSPACE_POSTFIX);
-
-    Mantid::Kernel::DateAndTime start = ws->run().startTime();
-    Mantid::Kernel::DateAndTime end = ws->run().endTime();
-
     const std::vector<Property *> &logData = ws->run().getLogData();
-
+    const TimeSeriesProperty<bool> *runningLog; // defined here to keep the
+                                                // object in scope. If
+                                                // applicable it is assigned a
+                                                // value within an if statement.
+
+    std::string running = "running"; // define the running log via a string to
```

Comment:
Do you mean like this: 

'namespace{
const std::string running = "running";
namespace MantidQt {
....code
}//close namespace 
`